### ISSUE TITLE
pubsub-javascript-sample using deprecated pubsub scope

### ISSUE THREAD

**Issue by aaronhauth** [twitchdev/pubsub-javascript-sample#3]
as an example for learning the twitch pubsub, the example is out of date. It utilizes chat_login, which is no longer available for new apps

**Comment by zbeucler2018** [comment]
Same here. Please update

**Comment by ickybodclay** [comment]
@aaronhauth @zbeucler2018 I don't think this repo is looked after, but if you check my PR #4 it fixes this issue.
